src/typescript/Scala.tmLanguage.ts

```diff
@@ -921,7 +921,7 @@ export const scalaTmLanguage: TmLanguage = {
       comment: 'For themes: Brackets look nice when colored.'
     },
     qualifiedClassName: {
-      match: '(\\b([A-Z][\\w]*))',
+      match: `(\\b([A-Z][\\w]*)(?:(?<=_)${opchar}+)?)`,
```

Comment:
Could there be multiple `_` or is only one allowed?

Reply:
The scanner only allows one `_`. 

See https://github.com/lampepfl/dotty/blob/master/compiler/src/dotty/tools/dotc/parsing/Scanners.scala#L1034

Reply:
Good, let's merge then!